src/Standards/Generic/Tests/CodeAnalysis/JumbledIncrementerUnitTest.1.inc

```diff
@@ -0,0 +1,83 @@
+<?php
+
+for ($i = 0; $i < 20; $i++) {
+    for ($j = 0; $j < 5; $i += 2) {
+        for ($k = 0; $k > 3; $i++) {
+            
+        }
+    }
+}
+
+for ($i = 0; $i < 20; $i++) {
+    for ($j = 0; $j < 5; $j += 2) {
+        for ($k = 0; $k > 3; $k++) {
+            
+        }
+    }
+}
+
+for ($i = 0; $i < 20; $i++) {
+    for ($j = 0; $j < 5; $j += 2) {
+        for ($k = 0; $k > 3; $j++) {
+            
+        }
+    }
+}
+
+for (; $i < 10; $i++) {
+    for ($j = 0;; $j++) {
+        if ($j > 5) {
+            break;
+        }
+        for (;; $k++) {
+            if ($k > 5) {
+                break;
+            }
+        }
+    }
+}
+
+for (; $i < 10; $i++) {
```

Comment:
Just a small note here: test 👍🏻 , but it might help to add a small comment above each test case saying what it is testing, so the test case on line 27 is the "okay" case for `for` loops with not all expressions set, while this test is the "flag me" case for `for` loops with not all expressions set.

Could possible also be done by renaming the `$i` variable to `$same` for this "flag me" code sample ? That should make it easier to spot.

Reply:
Oh and if you do change the variable name, I can imagine you may also want to do this for the pre-existing test cases.

Commit wise, would probably be good to do this in a separate commit before adding the extra test cases. (bug no biggie if you mix it in with this commit or do this in  a commit where you change it for the whole test case file in one go).

Reply:
Done! As you suggested, I added a commit before adding the extra test cases, renaming the variables in the pre-existing tests, and then another commit, renaming the variables in the new tests (which should be combined with the commit that adds the new tests before merging).